.github/workflows/nix-linux.yml

```diff
@@ -20,30 +20,10 @@ jobs:
       - run: nix-env -iA cachix -f https://cachix.org/api/v1/install
```

Comment:
btw! if you're DRYing these up, I think repetition of like the disk cleanup blocks could be factored out using like [yaml anchors](https://camel.readthedocs.io/en/latest/yamlref.html).

Reply:
another time